oxidation/libparsec/crates/client_connection/src/lib.rs

```diff
@@ -1,8 +1,9 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
 
-pub mod authenticated_cmds;
-pub mod client;
-pub mod command_error;
+mod authenticated_cmds;
+mod client;
+mod error;
 
-pub use authenticated_cmds::AuthenticatedCmds;
-pub use command_error::CommandError;
+pub use authenticated_cmds::{AuthenticatedCmds, API_VERSION_HEADER_NAME, PARSEC_AUTH_METHOD};
+pub use client::generate_client;
+pub use error::*;
```

Comment:
```suggestion
pub use error::{CommandError, CommandResult};
```

Limit the use of `glob` import it make the review harder and could cause hard to debug compile error.